src/screens/simulation/PublicSimulation.js

```diff
@@ -313,6 +323,27 @@ const getTimeframesFromValues = ({
     },
 ];
 
+const R0HelpIcon = (
+    <>
+        <span>R0</span>
+        <PopoverInfo
+            content={
+                <Paper style={{ padding: 10 }}>
+                    R0 correspond au nombre moyen de personnes infectées par une personne
```

Comment:
From what I've understood, we just need to get back to the description in the modal
(but this will change with the next PR)